pkgs/applications/networking/cluster/kubectl-node-shell/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenvNoCC, lib, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "kubectl-node-shell";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "kvaps";
+    repo = "kubectl-node-shell";
+    rev = "v${version}";
+    sha256 = "sha256-dAsNgvHgquXdb2HhLDYLk9IALneKkOxQxKb7BD90+1E=";
+  };
+
```

```suggestion
  strictDeps = true;
  buildInputs = [ bash ];
```

strictDeps is on by default when cross-compiling it is also good to set it in the native case

bash has to be added to buildInputs so that patchShebangs patches the shebang in the script

---

Thanks fixed :)